_src/dist/_layout.html

```diff
@@ -53,17 +53,17 @@
       <div class="container">
         <div class="row">
           <div class="col-md-6 col-sm-9">
-            <p>Open Budget: Oakland is an open-source project by<a href="http://openoakland.org">OpenOakland</a> to help citizens better understand Oakland's spending and budget process.</p>
+            <p>Open Budget: Sacramento is an open-source project by<a href="http://openbudgetsac.org/">Open Budget Sacramento</a> to help citizens better understand Sacramento's spending and budget process.</p>
             <ul class="list-inline">
-              <li><a href="http://openoakland.org"><img src="/images/global/openoakland-logo.png" width="150" alt="OpenOakland"></a></li>
-              <li><a href="https://twitter.com/openbudgetoak"><img src="/images/global/twitter-icon.png" alt="twitter"></a></li>
-              <li><a href="https://github.com/openoakland/openbudgetoakland"><img src="/images/global/github-icon.png" alt="github"></a></li>
+              <li><a href="http://openbudgetsac.org/"><img src="_src/images/global/codeforsac-logo.png" width="150" alt="Open Budget Sacramento"></a></li>
```

Comment:
Why did you change the path here?

Reply:
actually may not be needed since it is in `_src/dist`, we can review together.

Reply:
Ok, so I had a dumb issue with my repository. I had something like 5,000 staged commits, it was a mess and I might have messed a LOT of my directories trying to fix it. Could that be a reason why the paths are all messed up? It seems like the issue is repeating so I'm guessing it's an issue on my end.